docs/reference/math.md

```diff
@@ -196,10 +196,11 @@ expand the note below for further details.
 
 ### Interpolating functions
 
-#### `lerp` (C++20)
+#### `lerp` (C++20) {#lerp}
 
 !!! warning
-    `lerp`, based on [std::lerp], is only available for C++20 and later.
+    `lerp`, based on [std::lerp], is only available for C++20 and later.  For a pre-C++20 solution
+    for the special case where `t = 0.5`, see [`mean`](#mean) below.
```

Comment:
This reads a bit funny with the two "for"s in one sentence.  Maybe we don't need the redundant bit about C++20?

```suggestion
    `lerp`, based on [std::lerp], is only available for C++20 and later.  For the special case where `t = 0.5`, see [`mean`](#mean) below.
```